client/src/test/java/io/obswebsocket/community/client/message/event/highvolume/HighVolumeEventsSerializationTest.java

```diff
@@ -1,57 +0,0 @@
-package io.obswebsocket.community.client.message.event.highvolume;
-
-import io.obswebsocket.community.client.message.event.AbstractEventSerializationTest;
-import io.obswebsocket.community.client.message.event.highvolume.InputVolumeMetersEvent.InputLevels;
-import io.obswebsocket.community.client.model.SceneItem.Transform;
-import org.junit.jupiter.api.Test;
-
-class HighVolumeEventsSerializationTest extends AbstractEventSerializationTest {
-
-  private static final String TYPE = "highvolume";
-
-  @Test
-  void inputActiveStateChangedEvent() {
```

Comment:
shouldn't we move those in the right category instead of deleting them?

Reply:
I hadn't looked at the tests yet except to make stuff compile, was thinking on deciding that later :).

But it might be good to discuss whether to test the generated code at all. If a select few messages behave as expected I can pretty much guarantee that they all do. So what does testing every single message bring, other than more overhead when the protocol changes?
We could essentially generate the tests too, but I think that would add no additional value.

With generated code I think we should lean more on the e2e tests.

Reply:
I'm not entirely sure what the best-practices are for code generators. I'd expect that testing the generator would make sense. But any e2e tests that tests the generator output is essentially also an e2e test for the generator itself :).

Having units tests on top of e2e tests is generally preferred, and then testing the generator would make more sense than testing the output of the generator imo.